src/schema/v2/artist/helpers.ts

```diff
@@ -82,9 +82,18 @@ export const ARTIST_INSIGHT_MAPPING: Record<
     getLabel: () => "Active Secondary Market",
   },
   HIGH_AUCTION_RECORD: {
-    getDescription: (artist) => artist.highAuctionRecord,
-    getEntities: (artist) => artist.highAuctionRecord && [],
-    getLabel: () => "High Auction Record",
+    getDescription: (artist) =>
+      artist.highAuctionRecord &&
+      [
+        artist.highAuctionRecord.organization,
+        artist.highAuctionRecord.year,
+      ].join(", "),
+    getEntities: () => [],
+    getLabel: (artist) => {
+      return artist.highAuctionRecord
+        ? `High Auction Record (${artist.highAuctionRecord.price})`
+        : "High Auction Record"
+    },
```

Comment:
Note to self: `getEntities` apparently serves to determine whether or not the insight is included at all. Not sure any of that makes a lot of sense. Should consider restructuring this entire type.